README.md

```diff
@@ -121,6 +123,22 @@ This will generate a coverage.out file in the root directory of the repo which y
 $ go tool cover -html=coverage.out
 ```
 
+#### Code Organization
+All CLI go code lives under the "cmd" directory, who sub-directories map one is to one to the toplevel commands, namely
+addressable, command, db, device, ... version.
+The children of the above map to the sub-commands supported by each of these.
+Use the -f flag to provide file input, while n is reserved for names of objects.
+There are command specific and global flags such as verbose respectively. 
+Read the Cobra and Viper documentation for additional help.
+
```

Comment:
could we say something like `Use the global flag --help to get information about commands specific flags" ?

Reply:
This section is intended for the developer of the CLI, not the edgeX developer. Hummer's -- will change section title based on this comment

Reply:
Aha, I got your point! Now it make sense. 
If you have an idea how to state this more obviously ok, otherwise I am ok to resolve this conversation.